daemon/algod/api/server/v2/handlers.go

```diff
@@ -1227,7 +1215,7 @@ func (v2 *Handlers) GetApplicationBoxByName(ctx echo.Context, applicationID uint
 	}
 
 	response := generated.BoxResponse{
-		Name:  []byte(boxName),
+		Name:  boxName,
```

Comment:
Will this fix the base64 non-encoding that I identified elsewhere? I.e., is there some automatic process in the routing machinary that will know to encode it now that it is a string?

Reply:
@tzaffi `boxName` is already a `[]byte`.  The cast is superfluous.  My IDE alerted to me to the unnecessary cast.